### PR TITLE
Whitelist Kamen Rider Summonride Portal sys_usbd

### LINKED ISSUE #10694: [Feature Request] Whitelist Kamen Rider Summonride Portal USB device
Add support of Kamen Rider Summonride's USB peripheral in order to progress ingame.

The peripheral's ID is 
VID: 0E6F
PID: 200A

It's not currently whitelisted in RPCS3's sys_usbd

Currently, the game gets stuck here, asking for device connection. I've installed WinUSB drivers to the device and nothing is detected yet.
image

### PR DESCRIPTION
**Pull request by marcoluc97**
Should fix #10694

### CODE REVIEW

**Comment by marcoluc97** [comment]
Please @Megamouse review, thanks

File: rpcs3/Emu/Cell/lv2/sys_usbd.cpp [hunk 1]
```diff
@@ -175,7 +175,8 @@ usb_handler_thread::usb_handler_thread()
 
 		check_device(0x0E6F, 0x0241, 0x0241, "Lego Dimensions Portal");
 		check_device(0x0E6F, 0x0129, 0x0129, "Disney Infinity Portal");
-
+		check_device(0x0E6F, 0x200A, 0x200A, "Kamen Rider Summonride Portal");
+
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_usbd.cpp:179]: Why is there an extra line change? Are you using different line endings?

Comment (marcoluc97) [reply rpcs3/Emu/Cell/lv2/sys_usbd.cpp:179]: It is my fault I'm editing from a smartphone. If this need a change let me know.

Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_usbd.cpp:179]:
```suggestion

```

File: rpcs3/Emu/Cell/lv2/sys_usbd.cpp [hunk 2]
```diff
@@ -175,7 +175,7 @@ usb_handler_thread::usb_handler_thread()
 
 		check_device(0x0E6F, 0x0241, 0x0241, "Lego Dimensions Portal");
 		check_device(0x0E6F, 0x0129, 0x0129, "Disney Infinity Portal");
-
+		check_device(0x0E6F, 0x200A, 0x200A, "Kamen Rider Summonride Portal");
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_usbd.cpp:178]:
```suggestion
		check_device(0x0E6F, 0x200A, 0x200A, "Kamen Rider Summonride Portal");

```

**Pull request merged**